Fix scripts/generate-data.sh

AcKing-Sam opened the pull request:
Currently the address for github repo of ldbc_snb_datagen for hadoop has changed. The `generate-data.sh` now **redirects the address to ldbc_snb_datagen for spark instead of hadoop.** Therefore, I cannot download the correct package to generate data.

The latest repo address should be: https://github.com/ldbc/ldbc_snb_datagen_hadoop. And the v0.3.3 package can use `wget` to download, which is `wget https://github.com/ldbc/ldbc_snb_datagen_hadoop/archive/refs/tags/v0.3.3.tar.gz`. 
In the script, I use `${LDBC_SNB_DATAGEN_VERSION}` to represent the version number.

wey-gu commented:
Thanks for adapting this upstream change, besides, decoupling git with wget seems to be reasonable to me, too.

What do you think, please @HarrisChu

AcKing-Sam commented:
I am sorry for missing to change the variable in `scripts/env.sh` `LDBC_SNB_DATAGEN_VERSION`. I removed the `v` in it, and added the `v` in `scripts/generate-data.sh`. In this way, the tests should be passed.

wey-gu commented on scripts/env.sh:
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-LDBC_SNB_DATAGEN_VERSION=v0.3.3
+LDBC_SNB_DATAGEN_VERSION=0.3.3
```
shouldn't we keep this as is and remove the v from the url template in wget?

AcKing-Sam commented:
Yes, I intended to only modify `scripts/generate-data.sh`, while I found that the variable `LDBC_SNB_DATAGEN_VERSION` in `scripts/env.sh` was only used here. So I modify both `scripts/env.sh` and `scripts/generate-data.sh` and added `v` in front of the variable.

AcKing-Sam commented:
@HarrisChu Hello, in this issue, I try to update the `scripts/generate-data.sh`. 

To be specific, update the github repo address of `ldbc_snb_datagen_hadoop` because the original address did not work, which redirected to `ldbc_snb_datagen_spark`.

HarrisChu commented:
It's ok for me
